packages/sdk/contractkit/src/wrappers/transferownership.ts

```diff
@@ -0,0 +1,62 @@
+import { NetworkConfig, timeTravel } from '@celo/dev-utils/lib/ganache-test'
+import Web3 from 'web3'
+import { newKitFromWeb3 } from '../kit'
+import { AccountsWrapper } from './Accounts'
+import { Proposal, ProposalTransaction } from './Governance'
+
+// Implements a transfer ownership function using only contractkit primitives
+
+const expConfigGovernance = NetworkConfig.governance
+
+export async function assumeOwnership(web3: Web3, to: string, proposalId: number = 1) {
+  const kit = newKitFromWeb3(web3)
+  const ONE_CGLD = web3.utils.toWei('1', 'ether')
+  const accounts = await web3.eth.getAccounts()
+  let accountWrapper: AccountsWrapper
+  accountWrapper = await kit.contracts.getAccounts()
+  const lockedGold = await kit.contracts.getLockedGold()
+
+  try {
+    await accountWrapper.createAccount().sendAndWaitForReceipt({ from: accounts[0] })
+    await lockedGold.lock().sendAndWaitForReceipt({ from: accounts[0], value: ONE_CGLD })
+  } catch (error) {
+    console.log('Account already created')
+  }
+
+  const grandaMento = await kit._web3Contracts.getGrandaMento()
+  const governance = await kit.contracts.getGovernance()
+  const multiSig = await kit.contracts.getMultiSig(await governance.getApprover())
+
+  const tenMillionCELO = '10000000000000000000000000'
+
+  await lockedGold.lock().sendAndWaitForReceipt({ value: tenMillionCELO })
+
+  const ownershiptx: ProposalTransaction = {
+    value: '0',
+    to: (grandaMento as any)._address,
+    input: grandaMento.methods.transferOwnership(to).encodeABI(),
+  }
+  const proposal: Proposal = [ownershiptx]
+
+  await governance.propose(proposal, 'URL').sendAndWaitForReceipt({
+    from: accounts[0],
+    value: (await governance.getConfig()).minDeposit.toNumber(),
+  })
+
+  const tx = await governance.upvote(proposalId, accounts[1])
+  await tx.sendAndWaitForReceipt()
+  await timeTravel(expConfigGovernance.dequeueFrequency, web3)
```

Comment:
Does this file make sense to be in the wrappers folder? I feel that there should be a separate test-utils file or something. 

These `timeTravel` calls won't work as expected on live networks right? So I feel that it should be limited to just tests.